src/Microsoft.DotNet.Build.Tasks/AddDependenciesToProjectJson.cs

```diff
@@ -22,10 +22,12 @@ public class AddDependenciesToProjectJson : Task
         // Additional Dependencies to add to the project.json. May Optionally contain a version.
         // Will Override dependencies present in the project if there is a conflict.
         // AdditionalDependencies required metadata:  Name, Version
-
         [Required]
         public ITaskItem[] AdditionalDependencies { get; set; }
 
+        // Framework section which the additional dependencies apply to.  Empty is the default dependencies section.
```

Comment:
Unfortunately I think the assumption of "empty" means the default dependency section is not correct. For a project.json file the default dependencies section is a shared section between all the different target frameworks and our TargetGroup being empty may map to many different target frameworks so they aren't really equated.